### PR TITLE
Bug/sc 14810/horizontal scroll on homepage 2 step

### PR DESCRIPTION
**Pull request by pmanal1**
Completes [14810](https://app.shortcut.com/clarkcan/story/14810/horizontal-scroll-on-homepage-2-step).

Notes:
- Let me know if the looks should be changed.
- Confetti banner was sticking out, causing the horizontal scroll. It has been adjusted.
- Scroll buttons have been added to the topics list.

# Demo
https://user-images.githubusercontent.com/93054689/194635380-3443af1c-65f0-4353-b6d7-506bc3a211ec.mov

### CODE REVIEW

**Review by AStoerkel1 — APPROVED** [review]

File: src/app/cube/home/help/teach-now/teach-now.component.html [hunk 1]
```diff
@@ -1,11 +1,21 @@
 <div class="wrapper">
-    <div #topicScroll *ngIf="width > 750" class="topics">
-        <div class="shadow left" aria-hidden="true"></div>
-        <!-- NOTE: Add left and right buttons for scrolling if it becomes an issue -->
-        <button *ngFor="let topic of topics" class="topic-btn" (activate)="selectTopic(topic._id)" [ngClass]="{'selected': selectedTopic === topic._id}">{{ topic.name }}</button>
-        <span *ngIf="topics.length === 0">Loading Topics...</span>
-        <div class="shadow right" aria-hidden="true"></div>
+
+    <div *ngIf="width > 750" class="topics">
+        <button (activate)="horizontalScrollOnClick(false)">
```
Comment (mfranz1) [inline src/app/cube/home/help/teach-now/teach-now.component.html:4]: Let's remove the default styles on these buttons to match the other chevron's we have with no background and one of the clark greys as it's color

**Review by mfranz1 — CHANGES_REQUESTED** [review]

**Review by EVogel1999 — APPROVED** [review]

File: src/app/cube/home/help/teach-now/teach-now.component.html (hunk 1 above)
Comment (pmanal1) [reply src/app/cube/home/help/teach-now/teach-now.component.html]: I made the buttons the same as the topics buttons. Let me know if that doesn't work

Comment (mfranz1) [reply src/app/cube/home/help/teach-now/teach-now.component.html]: This works! @pzalep1 any thoghts?

**Review by mfranz1 — APPROVED** [review]

**Review by pzalep1 — APPROVED** [review]
Looks good! Thank you

**Pull request merged**